metalsmith/index.js

```diff
@@ -1,20 +1,32 @@
 const metalsmith = require('metalsmith');
 const layouts = require('metalsmith-layouts');
+const inPlace = require('metalsmith-in-place');
+const registerHelpers = require('metalsmith-register-helpers');
 
 metalsmith(__dirname)
-.source('../src') // Source directory
-.destination('../build') // Destination directory
-.clean(true) // Clean destination before
-.ignore(['app', 'assets', 'layouts', 'partials']) // Ignore other files than handlebars template
-.use(layouts({ // Wrap layouts around html
-	engine: 'handlebars', // Use the layout engine you like
-	rename: true,
-	directory: '../src/layouts',
+.source('../src')
+.destination('../build')
+.clean(true) // Clean destination before build
+.ignore(['app', 'assets', 'handlebars']) // Ignore other files than handlebars template
```

Comment:
Jep. Because you don’t want to ignore stuff because metalsmith handles all you files by default. 

But we handle most of the files with Gulp. So I had to ignore theme.

But this is obsolete anyways because of using gulpsmith 😏